docs/observabilityWalkthrough.md

```diff
@@ -0,0 +1,166 @@
+# Observability in Service Deployments
+
+Many Kubernetes deployments are composed of not just one, but many microservices, and this complexity is compounded by, for latency, scalability, and/or reliability concerns, that these microservices are often also deployed across multiple clusters as well.  This makes it hard to reason about the current state of any individual cluster -- and especially a collection of ones that all together constitute the workload.
+
+To help with this, Bedrock has a service introspection tool to provide better visibility the end to end deployment workflows. It integrates with the GitOps pipeline and service management that were setup in previous walkthrough guides. 
+
+The service introspection tool exists in two forms:
+1. Command line tool
+2. Web Dashboard
+
+The service introspection tool provides views into the current status of any change in the system, from continuous integration build to tracking the deployment of the container containing that commit in each of the downstream clusters consuming that container.
+
+The service introspection main components are:
+1. A Bedrock GitOps pipelines workflow. Currently supported in Azure DevOps
```

Comment:
We might explain each of these three in terms of their function, eg:

1. A Bedrock Gitops pipeline that reports back with telemetry data for each of the steps of the system.
2. An Azure Storage Table that stores all of the telemetry reported back.
3. Observability tools both for the command line and web dashboard